azext_edge/edge/providers/support/base.py

```diff
@@ -69,12 +69,17 @@ def process_crd(
         file_prefix = kind
 
     processed = []
-    namespaces = []
     for r in result.get("items", []):
-        if not namespace:
+        # Try to get namespace from metadata, if not found, use fallback_namespace if provided
+        try:
             namespace = r["metadata"]["namespace"]
-        namespaces.append(namespace)
+        except KeyError:
+            if fallback_namespace:
+                namespace = fallback_namespace
+            else:
+                raise ValueError("Namespace not found in CRD metadata and no fallback namespace provided.")
```

Comment:
Support bundle should not crash, but with the way the support bundle runner works I believe this exception becomes a logger .debug message, please confirm

Reply:
yes, will update!